hail/src/main/scala/is/hail/expr/ir/functions/Functions.scala

```diff
@@ -453,7 +453,12 @@ abstract class RegistryFunctions {
 
       override val returnType: Type = rType
 
-      override def returnPType(argTypes: Seq[PType], returnType: Type): PType = if (pt == null) PType.canonical(returnType) else pt(argTypes)
+      override def returnPType(argPTypes: Seq[PType], returnType: Type): PType = {
+        val rt = if (pt == null) PType.canonical(returnType) else pt(argPTypes)
+
+        // applySeeded sets EmitCode.m missing if any argument not required
```

Comment:
This is a misleading comment -- this requiredness is a property of the type, but `EmitCode.m` (missingness) is a property of the runtime value. Something like:
"the result of applySeeded is missing if any argument is missing" is correct.

Reply:
What does "result" refer to here? The return value of applySeeded is an EmitCode

Reply:
the emit code, yes

Reply:
the emit code is missing (m is true) if any argument is missing

Reply:
got it I think: the reader will know we're talking about the value present in the generated binary